packages/esm-patient-registration-app/src/config-schema.ts

```diff
@@ -155,6 +161,27 @@ export const esmPatientRegistrationSchema = {
         _default: [],
         _description: 'For coded questions only. Provide ability to add custom concept answers.',
       },
+      gender: {
+        _type: Type.Array,
+        _elements: {
+          value: {
+            _type: Type.String,
+            _description: 'The value for sex option',
+          },
+          label: {
+            _type: Type.String,
+            _default: null,
+            _description: 'The label displayed for sex option.',
+          },
+          id: {
+            _type: Type.String,
+            _default: null,
+            _description: 'The id for sex option.',
+          },
+        },
+        _default: [],
```

Comment:
@ibacher I think the gender field accepts a string, so it will store unknown/other sex options. The backend will support that.

Reply:
> Wouldn't it be better to just have the default values configured here instead of having special branching logic in the component? Also, have we confirmed that the configuration option is actually usable, i.e., will the backend accept it?
On making default values configured make sense, then one will only thin out the default values based on what one is interested in. I will go with that approach.